### PR TITLE
CSH-7816: create new component definition schema for showToolbars option

### PR DESCRIPTION
**Pull request by wwsno**
This option will allow to always show the toolbar of components. The default is to only show on hovering the mouse on the component.

### CODE REVIEW

File: lib/components-schema-v1_5_x.ts [hunk 1]
```diff
@@ -0,0 +1,596 @@
...
+                        description: 'names of properties this component can use',
+                    },
+
+                    selectionMethod: {
+                        enum: ['default', 'handle'],
+                        description:
+                            'How this component is selectable, by default the user can select inside the component to select it',
+                    },
+
+                    showToolbar: {
```
Comment (wwsno) [inline lib/components-schema-v1_5_x.ts:431]: This is the new option in the 1.5 version schema.

File: lib/models/components-definition.ts [hunk 2]
```diff
@@ -79,6 +79,13 @@ export interface Component {
      * How this component is selectable, by default the user can select inside the component to select it
      */
     selectionMethod?: 'default' | 'handle';
+
+    /**
+     * When to show the toolbar. By default only show on hovering the mouse on the component.
+     * Use the "always" option to show the toolbar in all cases.
+     */
+    showToolbar?: 'default' | 'always';
```
Comment (wwsno) [inline lib/models/components-definition.ts:87]: This is type annotation for usage in our Digital Editor code.

File: test/index.test.ts [hunk 3]
```diff
@@ -44,4 +48,14 @@ describe('getValidators', () => {
...
     });
+
+    it('should return amount of validators for version >= 1.4.0', () => {
+        const validators = getValidators('1.4.0', <any>null, <any>null, <any>null, <any>null, <any>null);
+        expect(validators && validators.length).toEqual(26);
+    });
+
+    it('should return amount of validators for version >= 1.5.0', () => {
+        const validators = getValidators('1.5.0', <any>null, <any>null, <any>null, <any>null, <any>null);
+        expect(validators && validators.length).toEqual(26);
```
Comment (wwsno) [inline test/index.test.ts:59]: There is one validator less because in 1.4 we dropped a validator related to a specific option of Adobe AEM.

**Review by wwjhu — APPROVED** [review]
👍

**Pull request merged**